e2e/whisper/whisper_mailbox_test.go

```diff
@@ -151,6 +152,77 @@ func (s *WhisperMailboxSuite) TestRequestMessageFromMailboxAsync() {
 	//assert
 	s.Require().NoError(err)
 	s.Require().Equal(1, len(messages.Result))
+
+	//check that there are no messages
+	resp = rpcClient.CallRaw(`{
+		"jsonrpc": "2.0",
+		"method": "shh_getFilterMessages",
+		"params": ["` + messageFilterID + `"],
+		"id": 1}`)
+
+	err = json.Unmarshal([]byte(resp), &messages)
+	//assert
+	s.Require().NoError(err)
+	s.Require().Equal(0, len(messages.Result))
+
+	//Request each one messages from mailbox, using same params
+	rpcClient.CallRaw(`{
+		"jsonrpc": "2.0",
+		"id": 2,
+		"method": "shh_requestMessages",
+		"params": [{
+					"peer":"` + mailboxPeerStr + `",
+					"topic":"` + topic.String() + `",
+					"symKeyID":"` + MailServerKeyID + `",
+					"from":0,
+					"to":` + strconv.FormatInt(time.Now().UnixNano(), 10) + `
```

Comment:
Shouldn't it be in seconds? `UnixNano()` returns nanoseconds.

Reply:
I don't found any information about it. But mailbox tests and wnode use same approach.

Reply:
It should be standardized and verified by unit tests then. As far as I remember, uint32 won't fit nanoseconds. I would change it to seconds so that everyone is on the same page.

Reply:
Nice catch! Fixed.